client/src/views/Simulation/components/ProvenanceGraph/ProvenanceGraph.vue

```diff
@@ -0,0 +1,87 @@
+<template>
+  <div class="provenance-graph-container" v-if="isOpen">
+    <aside slot="left">
+      <div class="btn-group btn-group-sm">
```

Comment:
Those would be better served as radio toggle:
https://getbootstrap.com/docs/4.3/components/buttons/#checkbox-and-radio-buttons

Reply:
Thanks for the suggestion, I've made a note to come back to this as polishing changes after I load some data in.